File: server/bookapp/models/book.py
from django.db import models

class Book(models.Model):
    isbn = models.IntegerField(unique=True)
    title = models.CharField(max_length=50)
    publisher = models.CharField(max_length=30)
    date = models.CharField(max_length=15)
    description = models.CharField(max_length=4000)
    pages = models.IntegerField()
    categories = models.CharField(max_length=200)
    ratingaverage = models.IntegerField()
    ratingnumber = models.IntegerField()
    thumbnail = models.CharField(max_length=200)
    bigthumbnail = models.CharField(max_length=200)

File: server/models/Book.js
class Book {
    constructor(isbn, title, publisher, date, description, pages, categories, ratingaverage, ratingnumber, thumbnail, bigthumbnail) {
        this.isbn = isbn;
        this.title = title;
        this.publisher = publisher;
        this.date = date;
        this.description = description;
        this.pages = pages;
        this.categories = categories;
        this.ratingaverage = ratingaverage;
        this.ratingnumber = ratingnumber;
        this.thumbnail = thumbnail;
        this.bigthumbnail = bigthumbnail;
    }
}

module.exports = Book;